packages/protocol/src/certificate/kinds/X509Base.ts

```diff
@@ -473,10 +473,19 @@ export namespace X509Base {
         return result;
     }
 
+    /** These extensions are minimum required for all Matter certificates. */
+    export const REQUIRED_PAA_EXTENSIONS = ["basicConstraints", "keyUsage", "subjectKeyIdentifier"];
```

Comment:
Given the way we're using this file maybe we could just call "X509" instead of "X509Base".  It's a namespace as much as a base class